test/csrc/sampler/test_random_walk.cpp

```diff
@@ -0,0 +1,18 @@
+#include <gtest/gtest.h>
+
+#include "../../../pyg_lib/csrc/sampler/random_walk.h"
+
+TEST(RandomWalkTest, BasicAssertions) {
+  const auto options = torch::TensorOptions().dtype(torch::kInt64);
+  const auto rowptr = torch::tensor({0, 2, 4, 6, 8}, options);
+  const auto col = torch::tensor({1, 3, 0, 2, 1, 3, 2, 0}, options);
```

Comment:
Could we abstract this into a `test_utils` module as a `simple_csr` to be reused in other tests?

Reply:
Sounds good :)